### PR TITLE
Web interface for the node

### PR DESCRIPTION
**Pull request by nicosantangelo**
### Web UI

The interface lists the users mined tiles with a link to visit them on the app. It also allows for a friendly UI for making RPC calls (works just as `./bin/cli` )

Run the server with:

```bash
$ cd browser
$ node server.js --apikey 'YOUR_API_KEY' --serverport 5000
```

Head over to http://localhost:5000 to see the UI

For development I recommend [relo](https://github.com/slezica/relo), and running it like this: `relo -g . -- node server.js --apikey 'YOUR_API_KEY'`, so the server gets automatically restarted on each change.

### To discuss

~1. I'm having some troubles getting the *mining speed*. I can't find a direct way to get the hashrate from the node without exposing it manually. Am I missing something?~ Adding features from https://github.com/decentraland/bronzeage-node/pull/63
2. Should I add something to the README or some other place in the docs?

### CODE REVIEW

**Comment by eordano** [comment]
Great work! Testing it now

**Review by eordano — APPROVED** [review]
LGTM

**Comment by martriay** [comment]
Looking at it!

**Comment by martriay** [comment]
A few comments:
- I think RPC input should be above output
- Current tiles view is not too friendly, can we distinguish the ones with content from the ones without and only show coords + content? Ej:
```
(0, -1) Empty (no link for this one)
(0, -2) Content
```
- Can we fix this log?:
image

**Comment by nicosantangelo** [comment]
- I think RPC input should be above output

👍 

- Current tiles view is not too friendly, can we distinguish de ones with content from the ones without and only show coords + content? Ej:
```
(0, -1) Empty (no link for this one)
(0, -2) Content
```

On it

- Can we fix this log?:
image

👍

**Comment by nicosantangelo** [comment]
@martriay Updated

File: browser/js/main.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        }
+      })
+      .catch(catchError)
+    })
+  }
+
+  function showTiles(tiles) {
+    var tilesHTML = '';
+    var EMPTY_TILE_CONTENT = new Array(63).fill('0').join('');
+    tiles[1].content = EMPTY_TILE_CONTENT + 'a'
```
Comment (martriay) [inline browser/js/main.js]: @NicoSantangelo :thinking:

Comment (nicosantangelo) [reply browser/js/main.js]: Sorry! forgot `-f` on the last local rebased commit

**Pull request merged**